cli/util/util.go

```diff
@@ -427,8 +427,12 @@ func IsRegularFile(filePath string) bool {
 
 // IsURL checks if str is a valid URL.
 func IsURL(str string) bool {
-	_, err := url.ParseRequestURI(str)
-	return err == nil
+	if !strings.HasPrefix(str, "http://") && !strings.HasPrefix(str, "https://") {
+		str = "http://" + str
```

Comment:
What about tcp:// ? Or any other scheme? It looks like there is should be a better logic.

And we need unit tests for the logic.

Reply:
here we supply a schema to make sure that this URL (it can be passed to localhost:50051) is separated from APP:INSTANCE_NAME.  This will be changed in the TNTP-1074 task.The test case added

Reply:
Done